### PR TITLE
LPS-175920 | Can break oneToMany relationship with custom object as parent in virtual instance

### PR DESCRIPTION
**Pull request by Hijky**
**Background**
Add a test case to break oneToMany relationship with custom object as parent in virtual instance

**Test Plan**
CanBreakManyToManyRelationshipWithSystemObjectAsParentInVirtualInstance
Given a virtual Instance
Given login with the new virtual Instance
Given a foundation object
Given manyToMany relationship foundationsOfUsers created
Given one foundation entry created
Given one userAccount entry created
Given I relate the userAccount entry with the foundation entry through the foundation PUT endpoint
When using the delete request deleteFoundation{relationshipName}
Then I can see no one entry correctly related in GET request response with the relationshipName=foundationsOfUsers

CanBreakOneToManyRelationshipWithCustomObjectAsParentInVirtualInstance
Given a virtual Instance
Given login with the new virtual Instance
Given a secondFoundation object
Given oneToMany relationship secondFoundationOfUsers created
Given one secondFoundation entries created
Given two userAccount entry created
Given all userAccounts are related to the secondFoundation through the secondFoundation PUT endpoint
When deleting one of the userAccount entries
Then I can see only one entry correctly related in GET request response

**JIRA Link:**
https://issues.liferay.com/browse/LPS-175920

### CODE REVIEW

**Comment by Hijky** [comment]
ci:test:sf

**Comment by Hijky** [comment]
ci:test:headless-functional:nocompile

**Comment by Hijky** [comment]
ci:test:sf

**Comment by Hijky** [comment]
ci:test:headless-functional:nocompile

**Review by magjed4289 — CHANGES_REQUESTED** [review]

File: modules/apps/object/object-rest-test/src/testFunctional/tests/BreakRelationshipBetweenCustomAndSystemObjects.testcase [hunk 1]
```diff
@@ -10,12 +10,25 @@ definition {
...
 		TestCase.setUpPortalInstanceNoSelenium();
 
+		task ("Given a virtual Instance") {
+			HeadlessPortalInstanceAPI.addPortalInstance(
+				domain = "liferay.com",
+				portalInstanceId = "www.able.com",
+				virtualHost = "www.able.com");
+		}
+
 		task ("Given a foundation object") {
```
[the remark this answers is not in the document]
Comment (Hijky) [reply modules/apps/object/object-rest-test/src/testFunctional/tests/BreakRelationshipBetweenCustomAndSystemObjects.testcase:20]: Except for  the tests about virtual instance, I think we need Foundation object and SecondFoundation object

**Review by magjed4289 — APPROVED** [review]

**Comment by Hijky** [comment]
ci:forward

**Pull request closed without merging**